### ISSUE TITLE
[A11Y] [iOS] Unable to change or Clear Date in the 'Date of Expiry' popup button.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#404]
**ENVIRONMENT:**
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

**PRE-REQUISITE:**
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

**STEPS TO REPRODUCE:**
1. Navigate to the URL: https://ampstart.com/
2. Navigate to "Components" link and double tap on it.
3. Navigate to the "Date of expiry" field in Text box section.
4. Try to Change the date of expiry and try to clear the values using Clear button.

**ACTUAL RESULTS:**
1. When the user changes the values in Date of expiry field and tries to select the values by double tapping, nothing happens. 
2. Double tapping on Clear button does not serve the intended purpose. Infact, double tapping on clear button does not perform any action at all.

**EXPECTED RESULTS:**
User should be able to change the Date of expiry values and be able to clear the values .

**ATTACHMENT:**
Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
Please see #415 for the date input accessibility discussion

**Comment by camelburrito** [comment]
This seems to be an issue with the browser impl. Nothing much to improve here
